model_compression_toolkit/gptq/README.md

```diff
@@ -0,0 +1,28 @@
+# Gradient-based Post-Training Quantization (GPTQ)
+
+The GPTQ package allows to run advanced quantization algorithm, in which the quantized parameters rounding is optimized after the initial quantization is applied.
+
+GPTQ preserves the principals of post-training quantization - it uses a small. unlabeled dataset, and does not require any additional information regarding the model.
+
+MCT provides several symmetric and uniform [trainable quantizers](../trainable_infrastructure/README.md) that allow to retrain the model. 
+
```

Comment:
MCT provides a variety of symmetric and uniform [trainable quantizers](../trainable_infrastructure/README.md) that allow you to retrain your model efficiently.